### PR TITLE
Fix vs std11 commands

### LINKED ISSUE #2630: Ctrl+; keybinding (search solution explorer) doesn't trigger with F# files are in focus
#### Repro steps

1. Create a new F# project.

2. Click in the editor on an F# project.

3. Press `Ctrl+;` keybinding.

#### Expected behavior

Focus is brought into the Solution Explorer search bar.

#### Actual behavior

Nothing happens.  Doesn't appear broken on C# or XAML.

search-solution-fails

#### Known workarounds

None.

#### Related information 

Latest F# nightly.

### PR DESCRIPTION
**Pull request by JoshVarty**
Fixes #2630 and #2335

The issue is that we were marking all commands as "handled" for `VSConstants.VsStd11` and `Guids.guidInteractive` command groups.

If we can't handle these commands we'll just let the others in the command chain have a chance to handle them.

Thanks to @majocha for help with the code.

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs [hunk 1]
```diff
@@ -36,15 +36,14 @@ type internal FsiCommandFilter(serviceProvider: System.IServiceProvider) =
 
     interface IOleCommandTarget with
         member x.Exec (pguidCmdGroup, nCmdId, nCmdexecopt, pvaIn, pvaOut) =
-            if pguidCmdGroup = VSConstants.VsStd11 then
-                if nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
-                elif nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteLineInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteLine null null
+            if pguidCmdGroup = VSConstants.VsStd11 && nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
+                do Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
```
Comment (saul) [inline vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs]: To answer your question on the issue ticket, the 'do' aren't necessary :)

Comment (JoshVarty) [inline vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs]: I'm brand new to F# so I'm willing to respect whatever coding conventions are in place for this repo. I've go no strong feelings one way or the other so I can remove them if people prefer.

Comment (saul) [inline vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs]: I can't speak for the actual maintainers of the code base, but idiomatic F# doesn't use 'do'. I'm not familiar with the VFT code base enough to state which is right but I've never seen 'do' used in production code at our work.

Comment (KevinRansom) [inline vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs]: @JoshVarty welcome to F# programming.

Please loose the do, in general match the style of the surrounding code is the watchword.  In this case the do is unnecessary. 

Kevin

Comment (smoothdeveloper) [inline vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs]: @JoshVarty just to give some context where do is used:

if you need some initialization code to run in constructor of types:

```fsharp
type MyType (myParameter) =
   do
      printfn "my parameter is: %s" myParameter
      // more code
   // rest of that type
```

also, can be useful to scope a disposable to a smaller scope than the function:

```fsharp
let myFunction () =
  let a = something ()
  do
      use willDisposeSoon = new SomethingDisposable()
      // ....
      // dispose called here
  doSomethingElse() |> ignore
  // some more code
```

In most other cases (I've encountered), it is mostly noise.

Comment (JoshVarty) [inline vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs]: :+1:

Thanks, I'll apply the change.

**Comment by KevinRansom** [comment]
@JoshVarty 

Thanks for this mate

Kevin

**Pull request merged**